website/scripts/ros/ros_traffic_signal.js

```diff
@@ -1,67 +1,102 @@
-/***
- * rostopic pub -r 10 traffic_signal_info cav_msgs/TrafficSignalInfoList  "traffic_signal_info_list: [{intersection_id: 0, state: 5,remaining_time: 1,lane_id: 0,remaining_distance: 1}]"
- */
+var unknown_intersection_id = -1; //Integer (0...65535)
+var unknown_signal_group = 0; //Integer (0...255). The value 0 shall be used when the ID is not available or not known.
```

Comment:
Should this be -1 like the intersection id?

Reply:
Signal group = 0 is unknow according to the J2735 doc